packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -208,10 +211,20 @@ class FlutterErrorDetails extends Diagnosticable {
     this.library = 'Flutter framework',
     this.context,
     this.stackFilter,
+    this.debugCreator,
     this.informationCollector,
     this.silent = false,
   });
 
+  /// Describer to gather important information about [debugCreator].
+  ///
+  /// This function is only called when [debugCreator] is not null.
+  ///
+  /// See also:
+  ///
+  ///  * [WidgetsBinding.initInstances], which binds this method.
+  static FlutterDebugCreatorDescriber describeCreator;
```

Comment:
why is this static? What if you have multiple kinds of libraries with different information they want to describe?